public/components/FeatureAnywhereContextMenu/AnywhereParentFlyout/AnywhereParentFlyout.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import './styles.scss';
+import { get } from 'lodash';
+import AddAnomalyDetector from '../CreateAnomalyDetector';
+
+const AnywhereParentFlyout = ({ startingFlyout, ...props }) => {
```

Comment:
Would still prefer to have some indication of what `props` is here. See comment here: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/482#discussion_r1195612865